Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCellView_Tests.swift

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerMentionCellView_Tests: XCTestCase {
+    /// Default reference width for the cell. This references
+    private static var defaultCellWidth: CGFloat = 374
+
+    func test_emptyAppearance() {
+        let view = ChatMessageComposerMentionCellView().withoutAutoresizingMaskConstraints
+        view.widthAnchor.constraint(equalToConstant: Self.defaultCellWidth).isActive = true
+
+        AssertSnapshot(view, variants: [.small])
+    }
+
+    func test_defaultAppearance() {
+        let view = ChatMessageComposerMentionCellView().withoutAutoresizingMaskConstraints
+        view.widthAnchor.constraint(equalToConstant: Self.defaultCellWidth).isActive = true
+
+        view.content = ("Mr Vader", "darkforce37", TestImages.vader.url, true)
+        AssertSnapshot(view, variants: [.small], suffix: "online indicator visible")
+
+        view.content = ("Mr Vader", "darkforce37", TestImages.vader.url, false)
+        AssertSnapshot(view, variants: [.small])
```

Comment:
With the new refactor you should now use `.onlyUserInterfaceStyle`. And don't forget to re-record the screenshots to update the names too 😛

Reply:
Thanks <3 Works flawlessly, now done :)